src/.vuepress/components/ExtensionsList.vue

```diff
@@ -5,19 +5,76 @@ The url argument must be of the form "https://paperback-ios.github.io/extensions
 -->
 
 <template>
-	<div>
-		<ul id="v-for-object">
-			<li v-for="extension in extensions"
-			:id="extension.name"
-			:key="extension.id"
-			>
-				{{ getName(extension.name) }}
-				<span v-for="tag in extension.tags" :key="tag.text"> <el-tag :type="tag.type" size="mini" effect="dark">{{ tag.text }}</el-tag> </span>
-			</li>
-		</ul>		
+	<div style="display: contents">
+		<tr>
+			<td>
+				<b>{{ name }}</b>
+			</td>
+			<td>{{ url }}</td>
+			<td><a :href="encodedURL()">Add to Paperback</a></td>
```

Comment:
The same layout could be used for the three items:
```suggestion
			<td>
				<b>{{ name }}</b>
			</td>
			<td>
				{{ url }}
			</td>
			<td>
				<a :href="encodedURL()">Add to Paperback</a>
			</td>
```
or
```suggestion
			<td><b>{{ name }}</b></td>
			<td>{{ url }}</td>
			<td><a :href="encodedURL()">Add to Paperback</a></td>
```

Reply:
IIRC its normal to inline the <td> tag so i think the latter would be good, I dunno how I miseed the name one